packages/side-nav/test/dom/side-nav-item.test.js

```diff
@@ -1,11 +1,20 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame, nextRender } from '@vaadin/testing-helpers';
+import sinon from 'sinon';
 import '../../src/vaadin-side-nav-item.js';
 import '@vaadin/icon';
 import '@vaadin/icons';
 
 describe('vaadin-side-nav-item', () => {
-  let sideNavItem;
+  let sideNavItem, baseUri;
+
+  beforeEach(() => {
+    baseUri = sinon.stub(document, 'baseURI').value('http://localhost/');
```

Comment:
**note:** web-test-runner adds a special query param with a session id to the base uri which causes the tests to break.